README.md

```diff
@@ -1,17 +1,12 @@
 # Static
 
-This application defines global templates for [GOV.UK](https://www.gov.uk) pages.
-
+This application defines global templates for [GOV.UK](https://www.gov.uk) pages. It is used in conjunction with [Slimmer](https://github.com/alphagov/slimmer), which is Rack middleware that takes a response from a Rails app and combines it with a template from Static.
 ## Screenshots
 
 `gem_layout` template:
 
 ![screenshot](/docs/gem_layout.png?raw=true)
```

Comment:
I found it quite useful; a picture speaks a thousand words (or at least a few paragraphs here).

I agree most apps don't need a screenshot as we can just point to a page which the app serves, but as this is a composite part in other pages, it's hard to know as an outsider _exactly_ what Static gives you. I like that this screenshot shows me at a glance that this particular template provides the cookie banner, header, feedback component and footer.

It's a shame it's so _big_ - I'd be tempted to move this towards the bottom of the README, so that people can still access it if they want, but it doesn't push important information below the fold. What do you think?

Reply:
> I found it quite useful; a picture speaks a thousand words (or at least a few paragraphs here).

It does if you know what you're looking for! Even as someone who knows what this app does, it takes me a moment to understand what it is I'm looking at. A more useful version of this image would have some annotations to explain what it's actually showing: the different sections of the page, where body content goes, customisations, etc.

An easy fix would be to just make it a link instead of an image, maybe as a sentence under "Live examples". I'd like to avoid leaving any of our READMEs with a "Screenshots" section, as it sets an unhelpful precedent.

An arguably better fix would be to guide the developer through getting the app running seeing the up-to-date version (as well as other templates) themselves, which I also suggested in [this other comment](https://github.com/alphagov/static/pull/2543#discussion_r666005881).

I'd be happy with just the easy fix though, since:

- Writing more documentation is out-of-scope for this work.
- Hopefully we won't have this app for _that_ much longer 🤞.

Reply:
Looks like we should remove the [Screenshots section from the README template](https://docs.publishing.service.gov.uk/manual/readmes.html#template-for-new-readmes) if we want to stop using it.

But I've changed it to a link and renamed the section to Live Examples :+1:

Reply:
Oh sheesh, I didn't know we had that manual. Thanks for pointing it out!

I've raised a PR to rewrite it so that we have one consistent template: https://github.com/alphagov/govuk-developer-docs/pull/3172

Anyway, the new section looks good, thanks for changing.